bootloader/grub.go

```diff
@@ -447,16 +462,19 @@ func staticCommandLineForGrubAssetEdition(asset string, edition uint) string {
 }
 
 var (
+	// Names of binaries depend on the target arch and are defined
+	// at grub_constants.go and grub_constants_{arch}.go
```

Comment:
No strong opinion, but if we ever need to cross reference the names for other arch during the build, perhaps it would be better to have:
```
type grubAssetNames struct {
    grubBinary string
    shimBInary string
}

var grubAssets = map[string]grubAssetNames {
    "amd64": {grubBinary: "grubx64.efi", shimBinary: "bootx64.efi"},
    "arm64": {grubBinary: "grubaa64.efi", shimBinary: "bootaa64.efi"},
}
```
and later in the code use:
```
import "github.com/snapcore/snapd/arch"

func getAssetsForArch() grubAssetNames {
    return grubAssets[arch.DpkgArchitecture())
}
```
and later on we can add overrides through env if needed, eg. UBUNTU_STORE_ARCH like one can do during prepare-image.